library/src/main/java/com/pokegoapi/api/gym/Battle.java

```diff
@@ -126,7 +122,7 @@ public Result start() throws LoginFailedException, RemoteServerException {
 	 */
 	public AttackGymResponse attack(int times) throws LoginFailedException, RemoteServerException {
 
-		ArrayList<BattleAction> actions = new ArrayList<BattleAction>();
+		ArrayList<BattleAction> actions = new ArrayList<>();
```

Comment:
There is no benefit to stripping the type declaration, but doing so could cause issues with some IDEs / compilers.